I18n Settings

MarceloWesley opened the pull request:
configure i18n for the project and the storybook

closes #5

MarceloWesley committed:
feat(i18n, storybook): add and configure i18n for the project and the storybook

closes #5

gabrielcdiniz commented on .storybook/preview.tsx:
```diff
@@ -0,0 +1,57 @@
+import i18n from '../src/i18n';
+import React, { Suspense, useEffect } from 'react';
+
+const withI18next = (Story, context) => {
+  const { locale } = context.globals;
+
+  useEffect(() => {
+    i18n.changeLanguage(locale);
+  }, [locale]);
+
+  return (
+    <Suspense fallback={<div>loading translations...</div>}>
+      <I18nextProvider i18n={i18n}>
+        <Story />
+      </I18nextProvider>
+    </Suspense>
+  );
+};
```
add "withI18next" to a new file in path ".storybook/decorators/i18n.tsx"

gabrielcdiniz commented on locales/pt.json:
```diff
@@ -0,0 +1,5 @@
+{
+
+    "Welcome to React": "Bem-vindo ao React"
+
+}
```
let an empty object

gabrielcdiniz commented on locales/en.json:
```diff
@@ -0,0 +1,5 @@
+{
+
+      "Welcome to React": "Welcome to React Dude"
+
+  }
```
let an empty object

gabrielcdiniz commented on src/App.tsx:
```diff
@@ -26,7 +29,7 @@ function App() {
       </div>
       <p className="read-the-docs">
-        Click on the Vite and React logos to learn more
+       {t('Welcome to React', {lng: 'en'})}
```
remove

MarceloWesley committed:
fix(code-review): fix code review

- let empty json in the locales folder
- add decorators folder in storybook
- delete the <p></p> tag in app.tsx
